src/Serilog/Debugging/LoggingFailedException.cs

```diff
@@ -20,6 +20,13 @@ namespace Serilog.Debugging;
 /// </summary>
 public class LoggingFailedException : Exception
 {
+    /// <summary>
+    /// Construct a <see cref="LoggingFailedException"/> to communicate a logging failure.
+    /// </summary>
+    public LoggingFailedException()
```

Comment:
Default exception constructors usually set a default message.

Reply:
why does an exception need a default constructor?

Reply:
I've never really liked the guideline, but this is the source of the convention: https://learn.microsoft.com/en-us/dotnet/standard/exceptions/best-practices-for-exceptions#include-three-constructors-in-custom-exception-classes